src/apprt/gtk/winproto/wayland.zig

```diff
@@ -57,6 +59,12 @@ pub const App = struct {
         registry.setListener(*Context, registryListener, context);
         if (display.roundtrip() != .SUCCESS) return error.RoundtripFailed;
 
+        if (context.kde_decoration_manager != null) {
+            // FIXME: Roundtrip again because we have to wait for the decoration
```

Comment:
This is fine for now. Ideally we can add the fd to the GTK event loop so the event loop wakes up when we have data to process.